problems/removeNumbersAtOddIndices.js

```diff
@@ -1,9 +1,16 @@
+const numberOfDigits = require("./numberOfDigits")
+const removeOddNumbers = require("./removeOddNumbers")
+
 /** Remove all numbers at odd indices from an array
 * @param {number[]} arr - The input array
 * @returns {number[]} - An array removing all elements initially appearing at an odd index
 */
-function removeNumbersAtOddIndices() {
-
+function removeNumbersAtOddIndices(arr) {
+    return arr.map((number, i)=>{
+        return i % 2 === 0 ? number : false 
+    }).filter(number =>{
+        return number !== false 
+    })
```

Comment:
Great work, look at the docs for the filter function, it can take in a optional second argument which becomes the index of the current accessed element and that could of been used to filter out the odd indices